packages/react-components/src/organisms/ManuscriptCard.tsx

```diff
@@ -283,38 +287,40 @@ const ManuscriptCard: React.FC<ManuscriptCardProps> = ({
                 }))}
               </StatusButton>
             </span>
-            <span css={buttonsContainerStyles}>
-              {isComplianceReviewer && (
-                <span>
-                  <Button
-                    primary
-                    small
-                    noMargin
-                    onClick={handleShareComplianceReport}
-                    enabled={canSubmitComplianceReport}
-                  >
-                    <span css={buttonStyles}>
-                      {complianceReportIcon} Share Compliance Report
-                    </span>
-                  </Button>
-                </span>
-              )}
-              {hasUpdateAccess && (
-                <span>
-                  <Button
-                    primary
-                    small
-                    noMargin
-                    onClick={handleResubmitManuscript}
-                    enabled={!!currentManuscriptVersion?.complianceReport}
-                  >
-                    <span css={buttonStyles}>
-                      {resubmitManuscriptIcon} Submit Revised Manuscript
-                    </span>
-                  </Button>
-                </span>
-              )}
-            </span>
+            {isActiveManuscript && (
```

Comment:
There's some stale state when we change manually to compliant / closed. We need to refresh to see the button disappear:
<img width="899" alt="Screenshot 2024-12-13 at 12 27 14" src="https://github.com/user-attachments/assets/987d2696-b7ac-4da8-b3f7-f8c41a3564d5" />

Reply:
I've just pushed an update to compute `isActiveManuscript` based on `selectedStatus` and not status